translations/en.json

```diff
@@ -636,7 +636,7 @@
             "description": "Internal and external users that have access to the link can download the item.",
             "generateLink": "Generate Link",
             "limitReached": "The maximum amount of public links has been reached",
-            "title": "Public links",
+            "title": "{{linkName}}s",
```

Comment:
title can be removed in favor of the plural of the link name. Another solution is to use a context of i18n here, to distinguish between the title of the public links for files, and the title for folders

Reply:
why should I remove the title? If I use the plural feature shouldn't be like this?
`"title": "{{linkName}}",
            "title_plural": "{{linkName}}s",`

And in the AddPublicLinkComponent.tsx `{t('publicLink.addLink.title', { count: 0, replace: { linkName } })}`